sentinel-2/ndvi_difference/README.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Normalized difference vegetation index difference between the latest acquisition and the acquisition 10-day prior to the latest one in a specified time interval
```

Comment:
I think this title is a bit too long, maybe change to

```suggestion
title: NDVI difference between two dates
```

Reply:
I also think this is too long. I shortened the title as you suggested